fake.py

```diff
@@ -285,7 +285,7 @@ def compose_frame(self, frame):
 
         if self.postprocess:
             mask = cv2.dilate(mask, np.ones((5,5), np.uint8) , iterations=1)
-            mask = cv2.blur(mask.astype(float), (10,10))
+            mask = cv2.medianBlur(mask, 5)
```

Comment:
I'm not sure how much it _really_ helps,  but i also replaced the blur on the mask to be median, since it generally handles edges better (and my wardrobe consists almost entirely of dark shirts) - but even i admit i'm not sure exactly how much it helped. There's probably a placebo effect going on.

Reply:
To be honest I didn't review the blurring code when it was merged in. Originally this repository was just a packaged-up version of that Google engineer's blog post. It did not come with blurring. 

I have recently taken a slightly better (stricter) approach to code review - I basically learnt this concept at work. I was a PhD student - we don't do code review.